game/script.rpy

```diff
@@ -1,10 +1,17 @@
 ﻿# The script of the game goes in this file.
 
-# Declare characters used by this game. The color argument colorizes the
-# name of the character.
+# Global variables
 
 define z = Character("???")
 
+transform chapter_fade:
+        xalign 0.5
+        yalign 0.5
+        alpha 0.08
+        easeout_back 2 alpha 0.9
+        pause 6
```

Comment:
Perhaps the pause for the chapter fade could be a bit shorter, down to 5 or even 4? I feel like the wait might be a _little_ bit too long